README.md

```diff
@@ -68,9 +78,9 @@ Select all that apply:
 ```
 [] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
```

Comment:
This is correct too.  It calls "innerHTML" on the first returned ".post" element.